src/coreclr/crossgen-corelib.proj

```diff
@@ -5,7 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="$(CoreClrProjectRoot)/tools/aot/crossgen2/crossgen2_inbuild.csproj" OutputItemType="Crossgen2" />
+    <ProjectReference Condition="'$(UseCrossArchCrossgen2)' != 'true'" Include="$(CoreClrProjectRoot)/tools/aot/crossgen2/crossgen2_publish.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Condition="'$(UseCrossArchCrossgen2)' == 'true'" Include="$(CoreClrProjectRoot)/tools/aot/crossgen2/crossgen2_crossarch.csproj" />
```

Comment:
Should we get rid of the `UseCrossArchCrossgen2` property, and build and use the cross-arch variant unconditionally? Most of the crossgen2 code is in libraries, building the .exe twice should not be a big deal.

If you adopt this suggestion, I think the original "inbuild" suffix would be more appropriate.

Reply:
We could -- the benefit would be that the AOT crossgen is much faster than coreclr one, so our build actually gets faster if we use the ILC copy.

Reply:
Eh, it's a few minutes, doesn't seem worth the complexity.